main.go

```diff
@@ -53,11 +53,13 @@ type Room struct {
 // User represents a user connected to the SSH server.
 // Exported fields represent ones saved to disk. (see also: User.savePrefs())
 type User struct {
-	Name     string
-	Pronouns []string
-	Bio      string
-	session  ssh.Session
-	term     *terminal.Terminal
+	Name           string
+	Prompt         string
+	FormattedPrompt string
```

Comment:
wait why is this exported. does it need to be saved?

Reply:
It doesn't needs to be exported (and it should not be as it needs to be recomputed depending on the user's color) but it needs to saved alongside the user's data for the time the user is logged in.

If you know a better way to do so I would love to know it as the implementation I made is sub-optimal.

Reply:
you can leave it unexported and it'll still be saved for the duration that the user's logged in: `formattedPrompt`

Reply:
Great tip. Thank you.